CMakeLists.txt

```diff
@@ -2,7 +2,8 @@ cmake_minimum_required(VERSION 3.21)
 
 project(vllm_extensions LANGUAGES CXX)
 
-option(VLLM_TARGET_DEVICE "Target device backend for vLLM" "cuda")
+# CUDA by default, can be overridden by using -DVLLM_TARGET_DEVICE=... (used by setup.py)
+set(VLLM_TARGET_DEVICE "cuda" CACHE STRING "Target device backend for vLLM")
```

Comment:
Could you describe the difference between this and what was there before?

Reply:
Accidentally committed, required to build locally, happy to put in a separate PR. The reason is that `option` only supports booleans, this is the equivalent syntax for a string variable.